README.md

```diff
@@ -17,19 +17,19 @@ The vizlab wiki at https://github.com/USGS-VIZLAB/vizlab/wiki has help pages on
 
 ##  Package management
   The Dockerfile has a few Docker-specific commands, but largely consists of shell commands to install packages that you have likely seen before.  The Dockerfile here starts with the [rocker geospatial image](https://hub.docker.com/r/rocker/geospatial), which already has R/Rstudio, geospatial libraries, and many standard packages already installed, so we really only need to add vizlab-specific packages on top of it.  You can go look at the Dockerfiles for the various rocker images to see exactly what is installed and from where.
-  R packages that come installed in the rocker images come from [MRAN's daily CRAN snapshots](https://mran.microsoft.com/documents/rro/reproducibility#snapshots),  corresponding to the date the rocker image was updated.  The `repos` option in R is already set to that same snapshot, so any package you install with `install.packages` without setting a repo will come from the same snapshot.  This obviously isn't the case for packages that are only on GRAN.  If you need specific other package versions, it is probably best to install from Github.  
+  R packages that come installed in the rocker images come from [MRAN's daily CRAN snapshots](https://mran.microsoft.com/documents/rro/reproducibility#snapshots),  corresponding to the date the rocker image was updated.  The `repos` option in R is already set to that same snapshot, so any package you install with `install.packages` without setting a repo will come from the same snapshot.  This obviously isn't the case for packages that are only on GRAN or other repositories, so any non-CRAN packages should be installed from GitHub with a release or commit specified.  Note that any package dependencies installed by `devtools::install_github` will still come from the same MRAN date as all the other packages.
```

Comment:
Oh, interesting! So you don't need to specify a tag in order to ensure that `devtools::install_github` sticks to whatever was available on the rocker image's date? But you can override it by specifying a tag?

Reply:
Only for _dependencies_ of a package installed viz `install_github`, since that uses the `repos` option (i.e. `getOption("repos")`) the same as `install.packages` does.  The actual package you specify in `install_github` is just coming from the github repo, so is not affected by the `repos` option.   Maybe that sentence should be clarified?

Reply:
Ohhhh, thanks. I definitely missed that, but you definitely already said it. I wonder if it _can_ be clarified for dummies like me...maybe with "Note that any dependencies of a package installed by...will still come from CRAN on the same MRAN date..."?